### PR TITLE
Updated statistics section

### PR DESCRIPTION
**Pull request by tommyod**
I read through and made some updates to the statistics section. Updated some of the code, and rewrote a few sentences.

### CODE REVIEW

File: packages/statistics/index.rst [hunk 1]
```diff
@@ -850,16 +850,17 @@ Can we conclude that education benefits males more than females?
 
 .. topic:: **Take home messages**
 
-   * Hypothesis testing and p-value give you the **significance** of an
-     effect / difference
+   * Hypothesis testing and the p-value give you the **significance** of an
```
Comment (GaelVaroquaux) [inline packages/statistics/index.rst]: I don't think that I agree with the addition of the "the". Maybe "p-values" (plural)?

**Comment by GaelVaroquaux** [comment]
Looks great! Thanks a lot. Only one minor comment on the "the".

**Comment by tommyod** [comment]
@GaelVaroquaux Cool! Thanks for the quick feedback. I agree with your comment and have addressed it :+1:

**Comment by GaelVaroquaux** [comment]
Darn, I just realized that your changes from pandas.tools.plotting to pandas.plotting require a recent version of pandas. We have a fixed set of dependencies, to ensure that the code is correct, and continuous integration on them, which failed.

We cannot merge this part of the change. As a matter of fact, we cannot merge if the continuous integration is not happy (as visible in the github PR interface).

**Comment by tommyod** [comment]
@GaelVaroquaux I see. That makes a lot of sense. I should have checked that before making changes. Thanks for the detailed feedback.

I have removed the changes to the code from this PR, but kept the changes to the writing if you want to merge those.

**Comment by vahtras** [comment]
I think it looks good, everything tests ok, no conflicts so I merge this

**Pull request merged**